tests/system/actions.go

```diff
@@ -13,13 +14,24 @@ type akResult struct {
 	returnCode int
 }
 
+func splitToArgs(cmdArgs string) []string {
+	re := regexp.MustCompile(`\w+=".*?"|\w+=[\w.:]+|"[^"]+"|[\w-.:/]+`)
```

Comment:
I think in proper regex you have to escape - and /
also, please have examples for this 3 cases 

```suggestion
	re := regexp.MustCompile(`\w+=".*?"|\w+=[\w.:]+|"[^"]+"|[\w\-.:\/]+`)
```

Reply:
Or maybe https://pkg.go.dev/github.com/google/shlex

Reply:
do we want to use archived unsupported package from 6-10 years go?

Reply:
@efiShtain I checked with regexp101  and tests and it seems there is no need in additional escaping

Reply:
> do we want to use archived unsupported package from 6-10 years go?

No commits != unsupported. But I'm OK with rolling our own.